build.sbt

```diff
@@ -28,3 +28,11 @@ libraryDependencies ++= {
 Revolver.settings
 
 javaOptions in Revolver.reStart += "-Dconfig.file=application.conf"
+
+// Copy over various properties
```

Comment:
ahhhh, I wasn't thinking straight when I commented on this. I thought it was copying properties from application.conf. Now it makes sense; please ignore.